Engine/DataFeeds/Enumerators/Factories/SubscriptionDataReaderSubscriptionEnumeratorFactory.cs

```diff
@@ -113,7 +113,7 @@ public IEnumerator<BaseData> CreateEnumerator(SubscriptionRequest request, IData
                 _factorFileProvider,
                 dataReader,
                 mapFileResolver,
-                _includeAuxiliaryData && CorporateEventEnumeratorFactory.ShouldEmitAuxiliaryBaseData(request.Configuration),
+                _includeAuxiliaryData,
```

Comment:
> _includeAuxiliaryData

We should remove this bool now that filtering happens higher in the stack?

Reply:
Gotcha 👌🏽